magento/adobe-stock-integration
# 1626: Modified date and time changes to original after Media Gallery synchronization

### Description (*)

This PR will introduce the fix to the issue of modified date changing back to original value after Media Gallery synchronization.
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration https://github.com/magento/adobe-stock-integration/issues/1626: Modified date and time changes to original after Media Gallery synchronization

### Manual testing scenarios (*)


1. Go to **Content - Media Gallery**
2. Select an image from Media Gallery, click on "three dots"
3. Select **Edit**
4. Change the image _Title_ by typing a new, valid text. Click **Save**
5. Click on "three dots" again and select **View Details**
6. Verify that the _Modified_ date and time has changed
7. Perform `php bin/magento media-gallery:sync`
8. Refresh the web page and View the image Details again
9. The modified date should not go back to its previous value.
